config/config.go

```diff
@@ -62,8 +65,6 @@ type ServerConfig struct {
 	AzureTenant           string `yaml:"azuretenant"`
 	Telemetry             string `yaml:"telemetry"`
 	HostNetwork           string `yaml:"hostnetwork"`
-	MQPort                string `yaml:"mqport"`
-	MQServerPort          string `yaml:"mqserverport"`
 	Server                string `yaml:"server"`
 	Broker                string `yam:"broker"`
```

Comment:
I believe there are duplicate fields in server config struct
what's the difference between `brokerendpoint` and `broker` fields?

Reply:
done